src/factory-daemon.js

```diff
@@ -23,7 +20,7 @@ class FactoryDaemon {
     if (options && options.type === 'proc') {
       throw new Error('This Factory does not know how to spawn in proc nodes')
     }
-    this.options = Object.assign({}, { type: 'go' }, options)
+    this.options = Object.assign({ type: 'go' }, options)
```

Comment:
Minor nit: we've required `merge-options` above, why not use it here instead of `Object.assign`?